js/chips.js

```diff
@@ -16,18 +17,47 @@ function selectChip(chip) {
 	selectedChip = chip; // set the newly-clicked chip to be the global selectedChip
 
 	return selectedChip; // return the freshly-set selectedChip variable back to the global scope
-	
+
 	// - get the ID of the current square that the selected chip is in
 	// - check the surrounding squares for available valid moves
 	// - call the validateMoves function, pass the selected chip and the current square as arguments.
 	//   might also need to pass-in the player or player's chip color
 	// validateMoves(selectedChip, currentSquare);
 }
 
+function selectSquare(square) {
+	//you can select a square only if you've already selected a chip
+	if (selectedChip) {
+		//only one square can be selected at a time. If one square already selected, clear previous selection.
+		if (selectedSquare) {
+			clearSquare();
+		}
+		//testing: log that square has been selected
+		console.log("Function selectSquare will select this " + square.classList[1])
+		//add class selected and some styling to the selected square
+		square.classList.add('selected');
+		//set the newly-clicked square to be the global variable of selectedSquare
+		selectedSquare = square;
+		//return the new selected square
+		return selectedSquare;
+	} else {
+		console.log("Choose a chip first!")
+		return false;
+	}
+}
+
+
 function clearSelection() {
-		selectedChip.classList.remove('selected'); // remove the `selected` class
-		selectedChip = undefined; // set the global variable back to undefined
-		return selectedChip; // return the undefined variable back to the global scope
+	selectedChip.classList.remove('selected'); // remove the `selected` class
+	selectedChip = undefined; // set the global variable back to undefined
+	return selectedChip; // return the undefined variable back to the global scope
+}
+
+function clearSquare() {
```

Comment:
This function is also essentially a copy of `clearSelection()`, but for squares.